### DISCUSSION TITLE
[委托]我的邦布经掉空洞里了

### DISCUSSION THREAD

**Discussion by LuYaoSEN** [share121/inter-knot#5857] [公告]
我的邦布掉空洞里了，你们谁能给我找到，事情完成后60个菲林和10000个丁尼

**Comment by HoshimiMiyabi120** [comment]
### 

> _**H·S·O·S·6以及治安局提醒您:**_
委托需谨慎，请下载治安局反诈app

**Reply by zean396** [reply]
布豪，绳网被渗透了🌝

**Comment by bxzwq** [comment]
这个委托我接不了

**Comment by Ropenetusers114514** [comment]
这个委托我接了

**Comment by qinYun256** [comment]
如果你家邦布的包装盒还在的话，你可以看一下包装盒最底部应该有一个说明书，这个说明书上应该会有关于帮助各种紧急情况的处理方法，你可以去看一看；如果买回来没有包装盒或者是不是原厂的话，那就当我没说吧（
